openslides_backend/migrations/migrations/0032_fix_motion_origin_relations.py

```diff
@@ -63,4 +70,26 @@ def migrate_event(
                         )
                     )
             return new_events
+        elif isinstance(event, UpdateEvent):
+            """Changes the update event
+            Precondition from backend: The backend sorts the events by type: create/update/delete
+            Therefore we don't need the self.get_additional_events-method and
+            can modify the original event.
```

Comment:
This might not hold true in case some other migration ran before, which modified the events and appended another one at the end. I'm not sure if we have such a migration, you might want to double check that.

Reply:
There are no motion-creates appended